src/main/java/uk/gov/ons/ctp/response/action/service/SocialActionProcessingService.java

```diff
@@ -0,0 +1,90 @@
+package uk.gov.ons.ctp.response.action.service;
+
+import com.godaddy.logging.Logger;
+import com.godaddy.logging.LoggerFactory;
+import java.util.List;
+import java.util.UUID;
+import org.springframework.beans.factory.annotation.Autowired;
+import org.springframework.beans.factory.annotation.Qualifier;
+import org.springframework.stereotype.Service;
+import uk.gov.ons.ctp.response.action.client.CaseSvcClientService;
+import uk.gov.ons.ctp.response.action.domain.model.Action;
+import uk.gov.ons.ctp.response.action.domain.model.ActionType;
+import uk.gov.ons.ctp.response.action.domain.repository.ActionCaseRepository;
+import uk.gov.ons.ctp.response.action.domain.repository.ActionRepository;
+import uk.gov.ons.ctp.response.action.domain.repository.ActionTypeRepository;
+import uk.gov.ons.ctp.response.action.message.ActionInstructionPublisher;
+import uk.gov.ons.ctp.response.action.message.instruction.ActionCancel;
+import uk.gov.ons.ctp.response.action.service.decorator.ActionAndActionPlan;
+import uk.gov.ons.ctp.response.action.service.decorator.ActionRequestDecorator;
+import uk.gov.ons.ctp.response.action.service.decorator.CaseAndCaseEvent;
+import uk.gov.ons.ctp.response.action.service.decorator.CollectionExerciseAndSurvey;
+import uk.gov.ons.ctp.response.action.service.decorator.SampleAttributes;
+import uk.gov.ons.ctp.response.action.service.decorator.context.ActionRequestContextFactory;
+import uk.gov.ons.ctp.response.casesvc.representation.CaseDetailsDTO;
+
+@Service
+@Qualifier("social")
+public class SocialActionProcessingService extends ActionProcessingService {
+
+  private static final Logger log = LoggerFactory.getLogger(SocialActionProcessingService.class);
```

Comment:
Does this class need any logging? If not, can the logger be removed here?

Reply:
Good spot. I will add logging for the cancel event.